namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-contacts-dev/resources/variables.tf

```diff
@@ -0,0 +1,77 @@
+variable "vpc_name" {
+  description = "VPC name to create security groups in for the ElastiCache and RDS modules"
+  type        = string
+}
+
+variable "kubernetes_cluster" {
+  description = "Kubernetes cluster name for references to secrets for service accounts"
+  type        = string
+}
+
+variable "application" {
+  description = "Name of the application you are deploying"
+  type        = string
+  default     = "HMPPS Challenge Support Intervention Plan"
```

Comment:
Wrong app name

Reply:
This is still showing the CSIP name.

Reply:
<img width="1061" alt="image" src="https://github.com/user-attachments/assets/218f371b-34f6-4400-b3d9-c0a08c430681">
have updated and pushed changes , no local changes to push , wonder if we are looking at the latest version here on github?

Reply:
it is updated here, https://github.com/ministryofjustice/cloud-platform-environments/pull/24710/files#diff-9c5b78f3c5764f34b40e66116e374643eeeb9d2f6cbad597febb057292fa9b05

Reply:
Yes - can see it now thanks. Maybe looking at an earlier commit.